src/OpenTelemetry.Contrib.Instrumentation.Runtime/RuntimeMetricsOptions.cs

```diff
@@ -0,0 +1,100 @@
+﻿// <copyright file="RuntimeMetricsOptions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenTelemetry.Contrib.Instrumentation.Runtime
+{
+    /// <summary>
+    /// Options to define the runtime metrics.
+    /// </summary>
+    public class RuntimeMetricsOptions
+    {
+        /// <summary>
+        /// Gets or sets a prefiix used for the metric names.
+        /// </summary>
+        public string MetricPrefix { get; set; } = "process_runtime_dotnet_";
```

Comment:
```suggestion
        public string MetricPrefix { get; set; } = "process.runtime.dotnet.";
```

Reply:
https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/runtime-environment-metrics.md#runtime-environment-specific-metrics---processruntimeenvironment

Reply:
Also, this need not be public.

Reply:
@cijothomas - I think whatever separator we use here should be the same one used in the instrument names for consistency? Also working set and cpu usage feel like outliers here. According to the naming specifications they would belong under process.cpu and process.memory

Reply:
@noahfalk / @cijothomas 
I almot sure that `process.runtime.dotnet` won't work for Prometheus, since the separator must be '_'
should  we do it configurable somehow?

Reply:
@omamoo I am not sure I follow fully.. The names would be as per the spec https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/runtime-environment-metrics.md#runtime-environment-specific-metrics---processruntimeenvironment - are you saying these names won't work for prometheus?

Reply:
@omamoo PrometheusExporter will do the conversion from `.` to `_`. It's not a concern of the instrumentation library nor the SDK, individual exporter will have to do the escape/mapping.

https://github.com/open-telemetry/opentelemetry-dotnet/blob/ff20a89a37d53b518acb62f73637cae84460ffaa/src/OpenTelemetry.Exporter.Prometheus/Implementation/PrometheusSerializer.cs#L177-L186

Reply:
@reyang 
Thanks for sharing. so  i thinks that `.` is the better choice too